### ISSUE TITLE
search/results.css not overriding from custom theme

### ISSUE THREAD

**Issue by rebeccaformfx** [nextcloud/server#6799]
Hey there!  I want to change the way the search app css is appearing, and I want to do it through my custom theme.  I have a css file under `path/to/my/theme/core/search/css/results.css` that has the appropriate classes to alter the styling, but when I look at the page, my styles aren't taking.  Looking at the console, the program is loading the old results.css file, and not my theme file.  Any pointers on how to override these styles within a theme, without editing the core files?

 
### Steps to reproduce
1.
2.
3.

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Server configuration

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)

**Updated from an older Nextcloud/ownCloud or fresh install:**

**Where did you install Nextcloud from:**

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

   "system": {
        "instanceid": "oc3yj61wfwou",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "ec2-18-220-180-252.us-east-2.compute.amazonaws.com",
            "demo.dais.pro"
        ],
        "datadirectory": "\/var\/www\/nextcloud\/data",
        "overwrite.cli.url": "http:\/\/ec2-18-220-180-252.us-east-2.compute.amazonaws.com\/nextcloud",
        "dbtype": "mysql",
        "version": "12.0.0.29",
        "dbname": "nextcloud",
        "dbhost": "localhost",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "theme": "dais",
        "maintenance": false,
        "enabledPreviewProviders": [
            "OC\\Preview\\PNG",
            "OC\\Preview\\JPEG",
            "OC\\Preview\\GIF",
            "OC\\Preview\\SVG",
            "OC\\Preview\\BMP",
            "OC\\Preview\\XBitmap",
            "OC\\Preview\\MP3",
            "OC\\Preview\\TXT",
            "OC\\Preview\\MarkDown"
        ],
        "mail_domain": "formfx.io",
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "mail_from_address": "info",
        "mail_smtphost": "smtp.gmail.com",
        "mail_smtpport": "587",
        "mail_smtpsecure": "ssl",
        "mail_smtpauth": 1
    }
}

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Insert your Nextcloud log here

include(): Failed opening 'layout.noscript.warning.php' for inclusion (include_path='/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Translation:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Util:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Nls:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Exception:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Date:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Stream_Filter:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Stream:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Socket_Client:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Stream_Wrapper:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Support:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Crypt_Blowfish:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Secret:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Text_Flowed:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Mime:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Mail:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Text_Filter:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Smtp:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Idna:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_ListHeaders:/var/www/nextcloud/apps/mail/vendor/pear-pear.horde.org/Horde_Imap_Client:/var/www/nextcloud/3rdparty/pear/console_getopt:/var/www/nextcloud/3rdparty/pear/pear_exception:/var/www/nextcloud/3rdparty/pear/archive_tar:/var/www/nextcloud/3rdparty/pear/pear-core-minimal/src:/var/www/nextcloud/apps') at /var/www/nextcloud/themes/dais/core/templates/layout.user.php#43
--




```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by nickvergessen** [comment]
Please provide the config report as well as the nextcloud.log as requested in the issue template

**Comment by rebeccaformfx** [comment]
Update with log and config report

**Comment by MorrisJobke** [comment]
Looks like this is fixed with #5070 in the upcoming Nextcloud 13
